packages/ui/text/src/lib/text.directive.spec.ts

```
@@ -31,13 +31,19 @@ describe('TextDirective', () => {
+        remove: () => {
+          console.log('Iancu');
+        },
         innerText: { trim: () => null },
       },
     } as ElementRef;
     const renderer2 = {} as Renderer2;
     const docuument = {
-      querySelectorAll: () => [{ remove: () => {} }],
+      querySelectorAll: () => [
+        {
+          remove: () => {},
```

ESLint

[error] Unexpected empty method 'remove'.

(@typescript-eslint/no-empty-function)